testing/cpm/cpm_generate_pins-simple-via-variable/verify/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#=============================================================================
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+cmake_minimum_required(VERSION 3.23.1)
+project(rapids-test-project LANGUAGES CXX)
+
+include(${rapids-cmake-dir}/cpm/init.cmake)
+include(${rapids-cmake-dir}/cpm/package_override.cmake)
+include(${rapids-cmake-dir}/cpm/detail/package_details.cmake)
+
+rapids_cpm_init()
+
+string(REPLACE " " ";" projects-to-verify "${projects-to-verify}")
```

Comment:
What is the purpose of this? Based on looking at https://github.com/rapidsai/rapids-cmake/pull/600/files#diff-f3fab95709f9de09a8138c45e47dd4135f671b6351c7df7b26f5ca82e2974c4dR29-R33, it seems like it should only be passing in a `;`-separated list.

Reply:
I am going to refactor the verify sub project across all tests and figure out if this is stil needed.

Reply:
Currently required since the list of values is passed as a string `"-DKEY=item1 item2 item3"`

Reply:
Where exactly is it being passed as a space-separated list? I don't see it.

Reply:
The add_custom_target was incorrectly quoting arguments. I have updated https://github.com/rapidsai/rapids-cmake/pull/601 with a corrected invocation that passes the list properly ( as `-D<key>="a;b"` )